frontend/src/components/dashboard/create/govsg/RadioChoice.tsx

```diff
@@ -26,7 +26,10 @@ function RadioChoice({
         <label htmlFor={id}>{label}</label>
       </div>
       {rest.checked && children && (
-        <div className={styles.body}>{children}</div>
+        <div className={styles.body}>{children[0]}</div>
+      )}
+      {rest.checked && children && children.length > 1 && (
+        <div className={styles.footer}>{children.slice(1)}</div>
```

Comment:
Sorry but could you help explain both conditions, I dont really understand? (Lines 28 and 31)

Reply:
My goal is to have the first child be part of body and for the rest to be part of footer.

iirc when I checked the usages of this component, there is either 1 or 2 children.

Reply:
I'll resolve this first.